test/train-sets/ref/help.stdout

```diff
@@ -224,6 +224,15 @@ Bootstrap:
   --bs_type arg         prediction type {mean,vote}
 Continuous actions tree with smoothing:
   --cats arg            number of tree labels <k> for cats
+  --min_value arg       Minimum continuous value
+  --max_value arg       Maximum continuous value
+  --bandwidth arg       Bandwidth (radius) of randomization around discrete 
+                        actions in terms of continuous range. By default will 
+                        be set to half of the continuous action unit-range 
+                        resulting in smoothing that stays inside the action 
+                        space unit-range:
+                        unit_range = (max_value - min_value) / num_actions
```

Comment:
Is num_actions equal to the "--cats" argument? Maybe we should avoid using both "actions" and "tree labels" to refer to the same thing? (If it is the same thing)

Reply:
yes it is good point, I'll use one term throughout